support/scripts/xen-guest

```diff
@@ -550,10 +555,10 @@ if [ ${OPT_GDBPORT} -eq 0 ]; then
 	printf '   GDB server:            %s\n' "${ARG_GDBPORT}"
 fi
 printf '\n'
-if [ ${OPT_GDBPORT} -eq 0 -a ${OPT_KERNEL} -eq 0 ]; then
+if [ ${OPT_GDBPORT} -eq 0 ] && [ ${OPT_KERNEL} -eq 0 ]; then
```

Comment:
Do you know why double quoting is not added here, as well?

Reply:
No idea, it did not complain about this line.

My hunch is that it sees it is checked as a number, so globbing/splitting would not make sense in this case.

Theoretically you can add quotes everywhere, just that in not all places it's necessary

Reply:
Seems to be less intelligent with `NICID` which is also a number only. 🤔

Reply:
We could just add quotes everywhere and be done with it :D

though I would prefer to do changes only to shut it up, cause things worked before anyways (apart from a few bugs that the check actually fixed)

Reply:
Is fine, I am testing some scripts and we merge this PR then.